doc/superset.md

```diff
@@ -0,0 +1,71 @@
+[<- Back to Index](index.md)
+
+# Apache Superset Demo
+
+SQL LRS's Postgresql support enables BI solutions like [Apache Superset](https://superset.apache.org/) to query the underlying SQL database. The SQL LRS project includes a Docker Compose file that can be used to demo this functionality.
+
+### Run the Docker Stack
+
+The Docker Compose stack includes an LRS, Postgresql, Keycloak, Superset and Redis (used as a cache by Superset). A Keycloak instance controls access to both the LRS and Superset.
+
+Note that for Keycloak to work (at least on Docker for Mac) you'll need to map your loopback IP to `host.docker.internal`. This can be done by adding the following line to your `/etc/hosts` file:
+
+    127.0.0.1 host.docker.internal
+
+To run the demo:
+
+    cd dev-resources/superset_demo
+    docker compose up -d
+
+or
+
+    make superset-demo
+
+Wait a minute or two for everything to start up.
+
+### Add xAPI data
+
+You'll need some data to visualize. Hook up the xAPI-conformant Learning Record Provider (LRP) of your choice or use [DATASIM](https://github.com/yetanalytics/datasim) to simulate learning activity.
```

Comment:
You should list the endpoint, API key, and API secret that should be passed to DATASIM here. In fact, I'd recommend that you include a CLI sample showing how you can use it to input statements into the LRS, in case any readers are unfamiliar with it.

Reply:
https://github.com/yetanalytics/lrsql/pull/231/commits/36b9cdb18316e0a86ad46f9e3d851cf26ad57ea3